voice/transcription.go

```diff
@@ -92,3 +92,16 @@ func (trans *Transcription) Contents(client *messagebird.Client) (string, error)
 	b, err := ioutil.ReadAll(resp.Body)
 	return string(b), err
 }
+
+// CreateTranscription creates a transcription request for an existing recording
+func CreateTranscription(client *messagebird.Client, callID string, legID string, recordingID string) (trans *Transcription, err error) {
```

Comment:
Why do we not need to set auth headers here? 

e.g. 

```
	req.Header.Set("Authorization", "AccessKey "+client.AccessKey)
	req.Header.Set("User-Agent", "MessageBird/ApiClient/"+messagebird.ClientVersion+" Go/"+runtime.Version())

```

Reply:
Is this done in the client ?

Reply:
This is used in another function (Contents) so not strictly in this PR changes.

I digged a bit in client code and the default `Accept` header is set to `application/json` and can't be overridden from outside: https://github.com/messagebird/go-rest-api/blob/a737548a56ce0bc1c6860aaed0b4e20fbfd29d26/client.go#L88

So for `text/plain` the author needed to make his/hers own http request, and hence the headers.